app/src/components/hoc/auth/AuthConfirmationPopover/index.js

```diff
@@ -52,10 +52,12 @@ export const AuthConfirmationPopover = ({
     <Popconfirm
       title={title}
       okText={okText}
-      cancelText={cancelText}
+      cancelButtonProps={{ style: { display: "none" } }}
+      okButtonProps={{ style: { margin: 0 } }}
```

Comment:
is override the inner classes for these buttons in a locally scoped css files?
This still seems like inline styles